Add js snippets to set textarea value.

ozakimami opened the pull request:
I would like to add set value to TextArea element.
Test case:https://app.autify.com/projects/111/results/1316004/capabilities/1553626/scenarios/4637454#
※Step5：Normal pattern、Step6：Error pattern(Cannot find the element with selector)

harokki commented:
@ozakimami 
Thank you!
The snippet `directory` is currently deprecated.
Please refer to the [README](https://github.com/autifyhq/autify-javascript-snippets#how-to-add-snippetsarticles) for instructions on how to add a new snippet.

tsuemura reviewed (changes requested):
In addition to the @harokki 's comment, please write a context about why the snippet is required. So that we can review it more easire and PM may notice a new feature from this snippet!

tsuemura commented on snippets/set-textarea-value.js:
```diff
@@ -0,0 +1,49 @@
+ */
+changeTextArea(element, setTextArea);
+
+function changeTextArea(input, value) {
```
nit: I assume the function's responsibility is to input a text. How about using the term instead of `change`?

```suggestion
function inputTextToTextArea(input, value) {
```

ozakimami commented:
I added the shipment to input Text to TextArea element.
We have already added the shipment  to input Text to Input element.
But , some customer want to input  Text to TextArea.

@harokki 
Thank you for your review comment!
I changed the directory and located to correct directory.(En and Ja)
Could you confirm ?

@tsuemura 
Thank you for your review!!
I agree with your comment, so change function name to ` inputTextToTextArea`;
Could you confirm?

ozakimami commented:
@harokki 
Should I change the page title of "要素に値を設定する" with it?（example:Input要素に値を設定する）

harokki commented:
@ozakimami 
> Should I change the page title of "要素に値を設定する" with it?（example:Input要素に値を設定する）

Yes, I think that is more clear.

ozakimami commented:
@harokki 
Update title for two documents, could you confirm?

harokki commented:
lgtm
